release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1233,6 +1267,8 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, when downloading a `kubeconfig` file for a ServiceAccount, an error was displayed and the ServiceAccount token was unable to be reached. This error was due to the removal of automatically generated secrets. With this update, the download `kubeconfig` action has been removed and the error no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-7308[*BZ#OCPBUGS-7308*])
 
+* Previously, if one or more existing plugins failed, the web console would not display a toast notification after an operator added a new plugin to the web console. This means that you would not receive a toast notification that prompts you to refresh the web console to view the new plugin. With this update, the web console checks when you added a plugin and then displays a toast notification on the console, regardless of any previously failed plugins. (link:https://issues.redhat.com/browse/OCPBUGS-10249[*OCPBUGS-10249*])
```

Comment:
I think you could just remove the second sentence in this.
```suggestion
* Previously, if one or more existing plugins failed, the web console would not display a toast notification after an operator added a new plugin.  With this update, the web console checks when you added a plugin and then displays a toast notification on the console, regardless of any previously failed plugins. (link:https://issues.redhat.com/browse/OCPBUGS-10249[*OCPBUGS-10249*])
```

Reply:
It's important to mention the reason for the notification in the context of this note. I did not remove that sentence.

Reply:
Another suggestion. Just thinking we should remove the "this means"

> Previously, if one or more existing plugins failed, the web console would not display a toast notification that prompts you to refresh the web console to view the new plugin after an operator added a new plugin to the web console.  With this update, the web console checks when you added a plugin and then displays a toast notification on the console, regardless of any previously failed plugins. (link:https://issues.redhat.com/browse/OCPBUGS-10249[*OCPBUGS-10249*])

Reply:
Thanks for the suggestion. The first sentence is over [32 words](https://www.ibm.com/docs/en/ibm-style?topic=medium-conversational-style) long. Restructured based on your example. 

How about:

Previously, if one or more existing plugins failed, the web console would not display a toast notification that prompted you to refresh the console. This action is required so that you can view a plugin after an operator adds the plugin to the console. With this update, the web console checks when the operator adds a plugin and then displays a toast notification on the console, regardless of any previously failed plugins. (link:https://issues.redhat.com/browse/OCPBUGS-10249[*OCPBUGS-10249*])